### ISSUE TITLE
architecture of the CNN

### ISSUE THREAD

**Issue by ruhollah22** [MuhammedBuyukkinaci/TensorFlow-Multiclass-Image-Classification-using-CNN-s#1]
Hi, Can u please tell me which architecture of CNN have you used in your code ? is it AlexNet or VGG16?

**Comment by MuhammedBuyukkinaci** [comment]
So sorry for being late. Forgive me. I didn't see your question. It has no specific architecture. If there should be a name, it may be called as BuyukkinaciNET, which is derived from my surname. If you have further questions, don't hesitate to ask.

**Comment by ruhollah22** [comment]
thanks for replying. My Final year project is too build a image classifiers.
will you answer few questions of mine?

**Comment by MuhammedBuyukkinaci** [comment]
Sure. What are your questions?

**Comment by ruhollah22** [comment]
is 4k-6k training dataset enough to get 0.1 error loss?
i did 2000  epochs before 1k epochs the error loss was decreasing decently but after that only 0.01 or 0.001 is decreasing after 1k epochs.

i checked the accuracy of training data set it shows 98% of accuracy while its training loss is near 0.515 n 0.614 . is it ok or i m doing something wrong.

**Comment by MuhammedBuyukkinaci** [comment]
There are some wrong parts in the project. There will be some updates that I will make on this project. Training data size isn't enough. You can increase it with web scraping.

**Comment by ruhollah22** [comment]
i m nt using ur architecture since it has many problems.

**Comment by MuhammedBuyukkinaci** [comment]
I accept your criticism. I will use Alexnet or VGG16 next times. Sorry for unreasonable architecture.

**Comment by MuhammedBuyukkinaci** [comment]
Hi @ruhollah22 . The architecture is updated. AlexNet is being used now. Sorry for being late.

**Comment by devdojo22** [comment]
@MuhammedBuyukkinaci Hello I hope you are doing fine.
Time flies. Sighs!!!! I was going through my old emails and I came across our convo back in 2018, I just wanted to know that my project was completed in 2018.

Thanks for the help back then.


Cheers
